Dockerfile

```diff
@@ -17,9 +17,9 @@ RUN apt-get update && \
   rm -rf /var/lib/apt/lists/*
 
 # install sonar-scanner
-ENV SONAR_VERSION="4.6.2.2472-linux"
-COPY sonar-scanner-cli-$SONAR_VERSION.zip /var/opt/
-RUN unzip /var/opt/sonar-scanner-cli-$SONAR_VERSION.zip -d /var/opt && \
+ENV SONAR_VERSION="5.0.1.3006-linux"
+RUN curl --insecure https://binaries.sonarsource.com/Distribution/sonar-scanner-cli/sonar-scanner-cli-$SONAR_VERSION.zip -o /var/opt/sonar-scanner-cli-$SONAR_VERSION.zip && \
```

Comment:
YOu can also remove `sonar-scanner-cli.zip` from this repo

Reply:
fixed